README.md

```diff
@@ -11,6 +11,7 @@ We will take small PRs and small features to this chart but more complicated nee
 ```yaml
 resourceGroup: "your application resource group"
 teamName: yourTeam
+secretNameSuffix: "Suffix to be added to secrets added by chart"
```

Comment:
I assume this isn't what you want? a space separated string?
does this need to be here by default?

and what would the suffix be?